aries-core/src/js/components/external/stories/Button.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Grommet, Box, Button } from 'grommet';
+import { aries } from '../../../../../../aries-site/src/themes/aries';
+
+export default {
+  title: 'Button',
+};
+
+export const Simple = () => (
+  <Grommet theme={aries} full>
+    <Box direction="row" gap="medium" justify="center" pad="large">
+      <Box
+        gap="medium"
+        align="center"
+        background={aries.global.colors['background-front'].dark}
+        pad="large"
+      >
+        <Button label="Default" />
+        <Button label="Anchor" href="#" />
+        <Button label="Primary" primary />
+        <Button label="Disabled" disabled />
+      </Box>
+      <Box
+        gap="medium"
+        align="center"
+        background={aries.global.colors.background.light}
```

Comment:
I'm not sure that hardcoding only `light` as an example is a good practice.

Reply:
Agreed. I am just pulling over what we had existing. Like I mentioned on the stand-up this morning, I think ideally each story example could be toggled between light and dark mode and so this wouldn't be needed. For the time being, I didn't want to spend too much time on the design on these stories so I just kept the light/dark explicitly coded like we had prior.